content/client-server-api/modules/third_party_networks.md

```diff
@@ -1,18 +1,18 @@
 
-### Third Party Networks
+### Third-party Networks
 
-Application services can provide access to third party networks via
+Application services can provide access to third-party networks via
 bridging. This allows Matrix users to communicate with users on other
 communication platforms, with messages ferried back and forth by the
 application service. A single application service may bridge multiple
-third party networks, and many individual locations within those
-networks. A single third party network location may be bridged to
+third-party networks, and many individual locations within those
+networks. A single third-party network location may be bridged to
 multiple Matrix rooms.
 
-#### Third Party Lookups
+#### Third-party Lookups
```

Comment:
I'm not sure "lookup" should be qualified this way. To me, "third-party lookup" is a lookup done by a third party; without a hyphen, it's a lookup of that third party.

Reply:
Aren't third-party lookups something that a Matrix user does to figure out what third-party locations (IRC rooms, Slack channels, etc.) exist for a given Matrix room?

This lookup isn't done by the third party - they don't know how to speak Matrix.

Reply:
The hyphenated version is correct here.  It is ambiguous, but hopefully context will make it clear.  If you want to blame it on English being a terrible language, then yes, it is.